### PR TITLE
[RFC] ADL-RVP: add definitions for RT711 2xRT1316 RT714 in SDW mode

### PR DESCRIPTION
**Pull request by libinyang**
RT711 is on link0
RT1316-1 is on link1
RT1316-2 is on link2
RT714 is on link3

Signed-off-by: Libin Yang <libin.yang@intel.com>

-------------
updated:
RT711 is on link0
RT1316-1 is on link1
RT714 is on link2
RT1316-2 is on link3

### CODE REVIEW

**Comment by plbossart** [comment]
@libinyang if this refers to the 3-in-1 board, this looks incorrect?

**Comment by libinyang** [comment]
> @libinyang if this refers to the 3-in-1 board, this looks incorrect?

@plbossart You are right. After checking the HW design, I confirmed that topology is wrong. I will update the patch and test the new topology tomorrow. Thanks for pointing it outl

**Comment by libinyang** [comment]
patch is updated to the new topology:
RT711 is on link0
RT1316-1 is on link1
RT714 is on link2
RT1316-2 is on link3

File: ADL-RVP/rt711-rt1316-rt714.asl [hunk 1]
```diff
@@ -0,0 +1,45 @@
+ /** @file
```
Comment (plbossart) [inline ADL-RVP/rt711-rt1316-rt714.asl]: would you mind renaming the file as adl_sdca_3_in_1 to match what the Linux PR does: 

https://github.com/thesofproject/linux/pull/2811/

**Review by plbossart — CHANGES_REQUESTED** [review]

File: ADL-RVP/rt711-rt1316-rt714.asl (hunk 1 above)
Comment (libinyang) [reply ADL-RVP/rt711-rt1316-rt714.asl]: @plbossart The patch is updated to rename to adl_sdca_3_in_1.asl

**Review by plbossart — APPROVED** [review]
thank you @libinyang

**Pull request merged**